File: Faltantes o por revisar/Angle.py
import math as math
class Point:
    def __init__(self, x, y):
        self.x = x
        self.y = y
 
    def equals(self, other):
        if (abs(self.x - other.x) < 1e-9 and (abs(self.y - other.y) < 1e-9)):
            return True
        return False
class Vec:
    def __init__(self, x, y):
        self.x = x
        self.y = y
def toVector(a, b):
    return Vec(b.x - a.x, b.y - a.y)
def angle(a, b, c):
    ba = toVector(b, a)
    bc = toVector(b, c)
    return math.acos((ba.x * bc.x + ba.y * bc.y) / math.sqrt((ba.x * ba.x + ba.y * ba.y) * (bc.x * bc.x + bc.y * bc.y)))


def DegToRad (d):
    return d * math.acos(-1.0) / 180.0


def RadToDeg(r): 
    return r * 180.0 / math.acos(-1.0)
puntos=[]
puntos.append(Point(0,-1))
puntos.append(Point(0,0))
puntos.append(Point(0,1))
res = angle(puntos[0], puntos[1], puntos[2]);
print(res)
print(RadToDeg(res))

File: Recien traducido/Geometria/Angle.java
//Dados 3 puntos A, B, y C, determina el valor del angulo ABC (origen en B) 
//en radianes. 
//Si se desea convertir a grados sexagesimales, 
//revisar degrees and radians.
import java.awt.Point;
public class Angle {

    static double angle(Point a, Point b, Point c) {
        Vec ba = toVector(b, a);
        Vec bc = toVector(b, c);
        return Math.acos((ba.x * bc.x + ba.y * bc.y) / Math.sqrt((ba.x * ba.x + ba.y * ba.y) * (bc.x * bc.x + bc.y * bc.y)));
    }

    static class Vec {
        public double x, y;
        public Vec(double _x, double _y) {
            this.x = _x;
            this.y = _y;
        }
    }

    static Vec toVector(Point a, Point b) {
        return new Vec(b.x - a.x, b.y - a.y);
    }

    static double DegToRad(double d) {
        return d * Math.PI / 180.0;
    }

    static double RadToDeg(double r) {
        return r * 180.0 / Math.PI;
    }

    public static void main(String[] args) {
        Point[] puntos = new Point[]{new Point(0, 0),
            new Point(1, 0), new Point(0, 1)};
        double res=angle(puntos[0], puntos[1], puntos[2]);
        System.out.println(res);
        System.out.println(RadToDeg(res));
    }

}
